code/modules/racial_roundstartperks/racial_perk.dm

```diff
@@ -0,0 +1,253 @@
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your body has been taxed to its limits, you need more time to recover before using this ability again."))
+		return FALSE
+	cooldown_time = world.time + 10 MINUTES
+	user.visible_message("<b><font color='red'>[user] lets out deep guttural growl as their eyes glaze over!</font><b>", "<b><font size='3px'><font color='red'>You abandon all reason as your sink into a blood thirsty frenzy!</font><b>", "<b><font color='red'>You hear a terrifying roar!</font><b>")
+	playsound(usr.loc, 'sound/voice/akularoar.ogg', 50, 1)
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("furiadol", 5)
+	return ..()
+
+/datum/reagent/stim/furiadol
+	name = "Furiadol"
+	id = "furiadol"
+	description = "A chemical naturally produced by several species that sends them into an all consuming frenzy."
+	taste_description = "bitterness"
+	reagent_state = LIQUID
+	color = "#5f95e2"
+	nerve_system_accumulations = 105 // so people can't spam it with others perks
+	addiction_chance = 100
+	appear_in_default_catalog = FALSE
+
+/datum/reagent/stim/furiadol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.stats.addTempStat(STAT_TGH, 35, STIM_TIME, "furiadol")
+	M.stats.addTempStat(STAT_ROB, 35, STIM_TIME, "furiadol")
+	M.stats.addTempStat(STAT_COG, -300, STIM_TIME, "furiadol")
+	M.stats.addTempStat(STAT_BIO, -300, STIM_TIME, "furiadol")
+	M.stats.addTempStat(STAT_VIG, -300, STIM_TIME, "furiadol")
+	M.stats.addTempStat(STAT_MEC, -300, STIM_TIME, "furiadol")
+
+/datum/reagent/drug/furiadol/withdrawal_act(mob/living/carbon/M)
+	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC, STIM_TIME, "furiadol_w")
+	M.stats.addTempStat(STAT_ROB, -STAT_LEVEL_BASIC, STIM_TIME, "furiadol_w")
+
+
+////////////////////////////////////////Naramad Based perks
+/datum/perk/overworkedspeed
+	name = "Sensorial Burst"
+	desc = "Several species are built for extreme speed, be it for charging forward and retreating back. But only due to a symbiotic relationship with fungal on your bones structures and muscles... making you expel left overs of its metabolism once a hour. "
+	active = FALSE
+	passivePerk = FALSE
+	var/cooldown = 60 MINUTES
+	var/initial_time
+
+/datum/perk/overworkedspeed/assign(mob/living/carbon/human/H)
+	..()
+	initial_time = world.time
+
+/datum/perk/overworkedspeed/on_process()
+	if(!..())
+		return
+	if(holder.species.flags & NO_BREATHE || holder.internal)
+		return
+	if(world.time < initial_time + cooldown)
+		return
+	initial_time = world.time
+	for(var/mob/living/carbon/human/H in viewers(5, holder))
+		if(H.stat == DEAD || H.internal || H.stats.getPerk(PERK_OVERWORKEDSPEED) || H.species.flags & NO_BREATHE)
+			continue
+		if(H.head?.item_flags & BLOCK_GAS_SMOKE_EFFECT || H.wear_mask?.item_flags & BLOCK_GAS_SMOKE_EFFECT || BP_IS_ROBOTIC(H.get_organ(BP_CHEST)))
+			continue
+
+		H.reagents?.add_reagent("amatoxin", 5)
+		H.emote("gasp")
```

Comment:
```suggestion
		H.emote("cough")
```

Reply:
The holder of the perk should give a span message in sight for everyone that they are releasing spores, too. Sort of like the anomaly that gives you the toxic fungus.